src/ctl/src/lib.rs

```diff
@@ -76,7 +76,10 @@ enum ComputeCommands {
 #[derive(Subcommand)]
 enum HummockCommands {
     /// list latest Hummock version on meta node
-    ListVersion,
+    ListVersion {
+        #[clap(short, long = "full-print", default_value_t = false)]
+        full_print: bool,
```

Comment:
nits: by convention, it is better call it `--verbose` (`-v`) to print more info.

Reply:
fix it